roles/sentry/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Clone latest sentry repo
+  ansible.builtin.git:
+    repo: https://github.com/getsentry/self-hosted.git
+    dest: "{{ homedir }}/sentry"
+
+
+- name: Copy sentry config
+  ansible.builtin.template:
+    src: sentry.config.tpl
+    dest: "{{ homedir }}/sentry/sentry/config.yml"
+    mode: 0644
+
+- name: Run sentry install/update script
+  ansible.builtin.command:
+    cmd: ./install.sh --no-report-self-hosted-issues --skip-user-prompt
+    chdir: "{{ homedir }}/sentry"
+  async: 900
+  poll: 10
+
+
+- name: Start services
```

Comment:
I added some async polling to the install steps to help the playbooks run consistently.  I found both the install script and `docker compose up` to take a v. long time (there's a bunch of images they build and then start up) and they can run longer than the standard ssh connection.  This causes a timeout and quiet failure when running the playbook and it's frustrating.  So I adjusted the `ansible.cfg` to add some performance improvements to the ssh connection ([useful link, tip #4](https://www.redhat.com/sysadmin/faster-ansible-playbook-execution)) and added these polling steps to help maintain the connection. It is slightly hacky, but ensures the role completes with these 10 minute install steps.

Reply:
I also added a `skip-user-prompt` flag to our `install.sh` invocation, otherwise the task hangs as it waits for a prompt to create our first user.  We can then create the user later in its own step (frustratingly not documented by sentry!)